package.json

```diff
@@ -6,7 +6,7 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "test:unit": "vue-cli-service test:unit --testPathPattern=."
+    "test:unit": "vue-cli-service test:unit --testPathPattern"
```

Comment:
Yes, this is OK (as discussed yesterday).

This allows a dev to type `npm run test:unit XYZ` to run just the unit tests for test pattern "XYZ".